File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentQuickCharge.java
package thedarkcolour.futuremc.enchantment;

public class EnchantmentQuickCharge extends EnchantmentCrossbowBase {
    public EnchantmentQuickCharge() {
        super(Rarity.COMMON, "quick_charge");
    }

    @Override
    public int getMinEnchantability(int enchantmentLevel) {
        return 5 + (enchantmentLevel - 1) * 7;
    }

    @Override
    public int getMaxEnchantability(int enchantmentLevel) {
        return 50;
    }

    @Override
    public int getMaxLevel() {
        return 3;
    }
}

File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentQuickCharge.kt
package thedarkcolour.futuremc.enchantment

class EnchantmentQuickCharge : EnchantmentCrossbowBase(Rarity.COMMON, "quick_charge") {
    override fun getMinEnchantability(enchantmentLevel: Int): Int {
        return 5 + (enchantmentLevel - 1) * 7
    }

    override fun getMaxEnchantability(enchantmentLevel: Int): Int {
        return 50
    }

    override fun getMaxLevel(): Int {
        return 3
    }
}